### ISSUE TITLE
Media-Time-Range scrub and skimming is broken if video is not pre-loaded, happens often on iOS 17.5

### ISSUE THREAD

**Issue by jelias** [muxinc/media-chrome#910]
just ran into another bug? seems that the media-time-range scrub and skimming is broken with the latest iOS update? Running iOS 17.5. It looks like it skims and updates, but one play it reverts back to the original playhead position?
Seems to be only an issue before the video starts play. And leaves the UI timecode vulnerable to being completely out of sync and just wrong

---

Some further research and testing from my end, it seems to happen most frequently when the video hasn't started loading for some reason. 

You can see it happen even on desktop in the media-chrome examples:
https://media-chrome.mux.dev/examples/vanilla/state-change-events-demo.html
1. Try to scrub
2. Hit play
3. Notice scrub goes back to 0.

**Comment by jelias** [comment]
> and to confirm, you weren't seeing this in iOS previously??

correct. in prior iOS versions this was not an issue. my guess is some sort of bug between loading the video player and the browser not letting it happen?

**Comment by luwes** [comment]
This has at least been partly fixed. We've successfully tested it on desktop and iOS 17 physical device.
On iOS 18 via Browserstack it was not working.
